core/src/main/java/io/opencensus/stats/View.java

```diff
@@ -37,7 +37,10 @@
   }
 
   /**
-   * Name of view. Must be unique.
+   * Name of view. Must be unique. Should be a ASCII string with a length no greater than 255
+   * characters.
+   *
+   * <p>Suggested format for name: {@code <web_host>/<path>}.
```

Comment:
See my next comment.  The detail that you added should probably go in the Name class.